### ISSUE TITLE
"Click here to view full recipe" and "Add to my recipes" are unnecessarily given as headings.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6474]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #AILimited; #A11yTCS; #Android; #AndroidPhone; #WCAG1.3.1; #GH_.NETMAUI_Android_Apr2022; #Headings​;

### Description

Talkback is announcing "Click here to view full recipe" and "Add to my recipes" unnecessarily as headings.

Similar issue observed in the following scenarios:

.NET MAUI<<My Recipes<<New: "Cancel" and "Save" are unnecessarily given as headings.
.NET MAUI<<My Recipes<<Edit: "Cancel", "Delete" and "Update" are unnecessarily given as headings.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Swipe till "search box" and enter recipe name and search.
3. "Search result" page will open.
4. Swipe till any recipe and double tap.
5. "Recipe" Screen will open.
6. Swipe till "Click here to view full recipe" and "Add to my recipes" and observe the issue.

https://user-images.githubusercontent.com/86710788/165446870-98ccab59-4ec9-4795-972a-34724c353f87.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
Talkback should not announce "Click here to view full recipe" and "Add to my recipes" unnecessarily as headings.
```

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang i have verified this issue in ".NET MAUI" App, Version 1.0.0
The issue is reproducing still. Can you please provide us version details in which these are fixed. 
Thanks.

**Comment by keerthiduvvuri** [comment]
Hi @rachelkang i have verified this bug. The issue is fixed.
I cannot close the bug, because this bug was logged by another tester. Can you close the bug from you'r end.
![Fixed_6474](https://user-images.githubusercontent.com/98728920/186148030-8d5b0c7c-73e1-464c-8e72-2d3165847b89.gif)
